src/ui/common/src/components/layouts/menuSidebar.tsx

```diff
@@ -182,162 +124,86 @@ const MenuSidebar: React.FC<{ user: UserProfile }> = ({ user }) => {
           component={RouterLink as any}
         >
           <img
-            style={{ maxWidth: '250px', width: '100%', marginLeft: '4px' }}
-            src="https://aqueduct-public-assets-bucket.s3.us-east-2.amazonaws.com/webapp/logos/aqueduct-logo-horizontal-light-transparent-resized.png"
+            src={
+              'https://aqueduct-public-assets-bucket.s3.us-east-2.amazonaws.com/webapp/logos/aqueduct_logo_color_on_white.png'
```

Comment:
TODO: @agiron123 find icon to be used here.

Also, @eunice-chan, do you have access to the S3 bucket which contains all of our assets?

Bucket Link:
https://s3.console.aws.amazon.com/s3/buckets/aqueduct-public-assets-bucket?region=us-east-2&tab=objects

Reply:
I have access to the S3 bucket!